app/database/user_item_store_compute_all_user_items.go

```diff
@@ -108,11 +108,12 @@ func (s *UserItemStore) ComputeAllUserItems() (err error) {
 							IFNULL(SUM(aggregated_children_attempts.tasks_solved), 0) AS tasks_solved,
 							IFNULL(SUM(aggregated_children_attempts.validated), 0) AS children_validated,
 							SUM(IFNULL(NOT aggregated_children_attempts.validated, 1)) AS children_non_validated,
-							SUM(items_items.category = 'Validation' AND IFNULL(NOT aggregated_children_attempts.validated, 1))
+							SUM(items_items_with_scores.category = 'Validation' AND IFNULL(NOT aggregated_children_attempts.validated, 1))
 								AS children_non_validated_categories,
 							MAX(aggregated_children_attempts.validated_at) AS max_validated_at,
-							MAX(IF(items_items.category = 'Validation', aggregated_children_attempts.validated_at, NULL)) AS max_validated_at_categories
-						FROM items_items
+							MAX(IF(items_items_with_scores.category = 'Validation', aggregated_children_attempts.validated_at, NULL))
+								AS max_validated_at_categories
+						FROM items_items AS items_items_with_scores
```

Comment:
I don't get this alias. At the scope you use it, it still have the score as you filter it inside this part of the query. 
It is like you would call `items` `items_with_score` while you filter it afterwards.

Reply:
fixed